reflex/model.py

```diff
@@ -43,12 +43,9 @@ def get_engine(url: Optional[str] = None):
         )
     # Print the SQL queries if the log level is INFO or lower.
     echo_db_query = os.environ.get("SQLALCHEMY_ECHO") == "True"
-    return sqlmodel.create_engine(
-        url,
-        echo=echo_db_query,
-        # Needed for the admin dash.
-        connect_args={"check_same_thread": False},
-    )
+    # Needed for the admin dash on sqlite.
+    connect_args = {"check_same_thread": False} if url.startswith("sqlite") else {}
```

Comment:
Just saw the original line was:
```python
connect_args={"check_same_thread": False} if conf.admin_dash else {}
```
But we don't have the admin_dash in config anymore. Should we still check that arg? I'm not sure what this param does actually.

Reply:
in sqlite, you get an exception if different threads use the same connection; and admin_dash runs on a different thread, so this param ignores that check.